docs/source/tutorial/index.rst

```diff
@@ -74,9 +73,9 @@ displayed in the console.
 
 .. code-block:: console
 
-  $ easylink run -p pipeline_demo_naive.yaml -i input_data_demo.yaml -e environment_local.yaml
+  $ easylink run -p pipeline_demo_naive.yaml -i input_data_demo.yaml
```

Comment:
Realized while making this PR that `environment_local` isn't even necessary -- it is the default!